ERCS/erc-7845.md

```diff
@@ -0,0 +1,644 @@
+
+#### Transaction
+
+```typescript
+interface Transaction: {
+    to: Destination; // REQUIRED
+    chainId: number; // REQUIRED
+    amount: (number|string); // REQUIRED
+    calldata: string; // OPTIONAL
+}
+```
+
+The above `Transaction` interface is a transaction definition that allows a wallet to perform their solution to a problem. There may be 1 or more transactions for a `Solution`.
+
+- The `to` property is `REQUIRED`, takes a `Destination` type and represents the target for this Solution.
+- The `chainId` property is `REQUIRED`, takes a number and represents the chain that this `Transaction` is targeted at. The `chainId` is `REQUIRED` here because the wallet `MUST` know where to send assets from as an origin due to the existence of multichain assets.
+- The `amount` property is `REQUIRED` and represents the amount to be sent as part of the `Transaction`. Accepts either a `number`, which represents an ether unit, or a `string` which can be used for BigNumbers.
+
+## Rationale
+
+- Uses the Ethereum JSON-RPC JSON wrapper for greater compatibility.
+- The interface definitions use only generic primitive types to ensure wide compatibility for any programming language.
+- The interface definitions defined in this ERC attempts to cover as many scenarios as possible, from an Orchestrator perspective that a wallet may ask for, but focuses on core blockchain functionality.
+- Certain high-level definitions, such as the `Problem` object definition, are sparse by design to allow space for future features introduced by other ERC's or network upgrades.
+- Terminology is targeted towards a non-technical lexicon to aid in wider adoption and understanding.
+- Nearly all options are `REQUIRED`, `SHOULD` and `OPTIONAL` to allow for both wallet and Orchestrator flexibility in providing solutions for the wallet request.
+- It's understood that the Orchestrator interpretations and implementations will vary, so where possible the specification enforces `REQUIRED` and `MUST` to provide a minimal level of service to an end-user or another service.
+- The specification is **NOT** intended to standardise or modify the internal data structure or communication layer of an Orchestrator.
+- Other parameters that could be considered, such as gas limits and estimations, are delegated back to the wallet as ultimately it is the wallet that will execute the solution(s).
+
+## Backwards Compatibility
+
+No backward compatibility issues found.
+
+## Reference Implementation
+
+### Example requests and responses for solutions
+
+The following examples show a few common scenarios with their requests to, and from, an Orchestrator. All examples are chain abstracted (ChA¹) by default, unless specified.
+
+#### Sending an [ERC-20](./eip-20.md) token to another address
+
+The following request performs an action:
+
+- from 0xbafB4E1EFA94B359e2E175CF6156AedA2cACa165
+- towards address 0x50840CE036eEf2005d3c4d6f6Eb65f8116a01629
+  - with symbol USDC, amount 5
+
+:::info
```

Comment:
I'm not sure this'll render the way you want. Double check it after merging.

Reply:
Sorry you're right i was using a reference from elsewhere. I have now changed it to the alert standard here: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts